components/display/index.rst

```diff
@@ -134,19 +134,20 @@ specific sizes, with ESPHome you have the option to use **any** TrueType (``.ttf
 at **any** size! Granted the reason for it is actually not having to worry about the licensing of font files :)
 
 To use fonts you first have to define a font object in your ESPHome configuration file. Just grab
-a ``.ttf`` file from somewhere on the Internet and create a ``font:`` section in your configuration:
+a ``.ttf`` file from somewhere on the internet and upload it to the ESPHome root folder.
```

Comment:
```suggestion
a ``.ttf`` file from somewhere on the internet and place, for example,
place it inside a `fonts` folder next to your configuration file.
```

Reply:
`place` is double here?
> place, for example, place

==> ... and place it e.g., inside a `fonts` folder ...

Reply:
right 🤕 
Fixed 🩹